app/views/courses/show.html.erb

```diff
@@ -73,18 +73,18 @@
 <div class="course-tabs govuk-tabs">
   <h2 class="govuk-tabs__title">Contents</h2>
 
-  <ul class="govuk-tabs__list">
+  <ul class="govuk-tabs__list" role="tablist">
```

Comment:
I don't think this role is needed, it's not included in the design system, see also: https://simplyaccessible.com/article/danger-aria-tabs/